src/gemdat/jumps.py

```diff
@@ -227,13 +227,15 @@ def collective(self, max_dist: float = 1) -> Collective:
 
     @weak_lru_cache()
     def activation_energies(
-            self) -> dict[tuple[str, str], tuple[float, float]]:
+            self, n_parts: int) -> dict[tuple[str, str], tuple[float, float]]:
```

Comment:
Is activation_energies not better returned as a dataframe? Feel free to make a new issue for this

Reply:
Yes, see #249